en/docs/quick-starts/attribute-config-sample.md

```diff
@@ -68,18 +68,21 @@ If you have your own application, click the button below.
 
 3.  Enter the **Client ID** and **Callback URL**.
 
-    ![playground-login](../../assets/img/samples/playground-login.png)
-
 4.  Click **Authorize**. 
 
 5.  Give the consent to share your attribute information with the application by clicking on **Continue**. 
 
-    ![consent-playground](../../assets/img/samples/consent-playground.png)
+    <img src="../../assets/img/samples/consent-playground.png" alt="alt text" width="400" height="500">
 
 6.  Enter the **client secret** and the **callback url**. 
 
 7.  Click **Get access token**. 
 
 8.  You are now successfully logged in to the sample application. You can see the values of the local attributes that were configured. You can also see the field, **Subject Claim URI** as **sub**, which contains the value of the local attribute you configured as the subject attribute URL. 
 
-    ![playground-sub](../../assets/img/samples/playground-sub.png)
+    <img src="../../assets/img/samples/playground-sub.png" alt="alt text" width="500" height="500">
+
+!!! info "Related topics" 
+    -   [Attribute Management](TODO:insert-link-to-concept)
+    -   [Request attributes](../../../guides/login/request-app-claims)
```

Comment:
Please change to the format:

Concept : xxxx
Guide: xxx

please check this on other pages as well